.changeset/spicy-birds-remember.md

```diff
@@ -0,0 +1,14 @@
+---
+'@commercetools-docs/gatsby-theme-api-docs': major
+'@commercetools-docs/gatsby-theme-code-examples': major
+'@commercetools-docs/gatsby-theme-constants': major
+'@commercetools-docs/gatsby-theme-docs': major
+'@commercetools-docs/gatsby-transformer-mdx-introspection': major
+'@commercetools-website/api-docs-smoke-test': major
+'@commercetools-website/docs-smoke-test': major
+'@commercetools-website/site-template': major
+---
+
+Migrate packages to Gatsby `v3`. Please refer to the official release notes: https://www.gatsbyjs.com/docs/reference/release-notes/v3.0/
+
+Note that in the docs-kit packages there are no breaking changes.
```

Comment:
Curiosity: Why do the kit packages then need to get a major bump? 

It's the right communication to users nevertheless.

Reply:
> Why do the kit packages then need to get a major bump?

Because of the peer dependencies of gatsby